src/view/dataElements/xdmObject.styl

```diff
@@ -15,7 +15,3 @@ governing permissions and limitations under the License.
 .XdmObject-treeContainer {
   width: 300px;
 }
-
-.XdmObject-description {
-  font-size: 18px;
-}
```

Comment:
This is unrelated to the rest of the PR. I noticed it should be in noSelectedNodeView.styl.